CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 
 [1]: https://www.npmjs.com/package/@google-cloud/firestore?activeTab=versions
 
+## [2.0.0](https://www.github.com/googleapis/nodejs-firestore/compare/v1.3.0...v2.0.0) (2019-05-14)
```

Comment:
Can we more explicit about dropping Node 6 support? Something like:

```

**Note:** As of v2.0.0, Firestore requires at least Node v8.10.0.

### Bug Fixes...

```

I am not 100% that everyone will understand the meaning of the engine field.